Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/FlyoutSamplePage.xaml

```diff
@@ -118,51 +118,101 @@ turpis semper nisi maecenas, nascetur dictumst sed arcu aenean varius dis leo ha
 						</smtx:XamlDisplay>
 
 						<!-- Material MenuFlyout + MenuFlyoutSubItem -->
-						<smtx:XamlDisplay UniqueKey="FlyoutSamplePage_Material_Flyout"
-										  smtx:XamlDisplayExtensions.Header="Nested MenuFlyout">
+						<win:Grid>
```

Comment:
This will hide it for everything except windows, but I think it works on wasm ?

/cc @kazo0